custom_components/tahoma/select.py

```diff
@@ -1,27 +1,36 @@
 """Support for Overkiz select devices."""
-from homeassistant.components.cover import DOMAIN as COVER
 from homeassistant.components.select import SelectEntity
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
-from custom_components.tahoma.coordinator import OverkizDataUpdateCoordinator
-from custom_components.tahoma.cover_devices.tahoma_cover import (
-    COMMAND_CLOSE,
-    COMMAND_OPEN,
-)
-
 from .const import DOMAIN
-from .entity import OverkizEntity
-
-CORE_OPEN_CLOSED_PEDESTRIAN_STATE = "core:OpenClosedPedestrianState"
-COMMAND_SET_PEDESTRIAN_POSITION = "setPedestrianPosition"
+from .entity import OverkizDescriptiveEntity, OverkizSelectDescription
 
-OPTION_TO_COMMAND = {
-    "closed": COMMAND_CLOSE,
-    "open": COMMAND_OPEN,
-    "pedestrian": COMMAND_SET_PEDESTRIAN_POSITION,
-}
+SELECT_DESCRIPTIONS = [
+    OverkizSelectDescription(
+        key="core:OpenClosedPedestrianState",
+        name="Position",
+        icon="mdi:content-save-cog",
+        options=["closed", "open", "pedestrian"],
+        select_option=lambda option, execute_command: execute_command(
+            {
+                "closed": "close",
+                "open": "open",
+                "pedestrian": "setPedestrianPosition",
+            }[option]
+        ),
+    ),
+    OverkizSelectDescription(
+        key="io:MemorizedSimpleVolumeState",
+        name="Memorized Simple Volume",
```

Comment:
What is it?

Reply:
Like a My position; but for the audio volume